app/views/users/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Here is a specific user</h1>
\ No newline at end of file
+  <%= render partial: "shared/person_info" %>
+  
+  <section class="bio_section">
+    <h1 class="bio_header">Bio</h1>
+    <p class="bio_paragraph">
+      <%= @user.bio %>
+    </p>
+  </section>
+  
+  <section class="posts_section">
+    <% @user.posts.order(created_at: :asc).limit(3).each_with_index do |post, i| %>
```

Comment:
- [ ] The posts that are rendered on the user show page should be the ones gotten from the `recent_posts` method in the User model. Kindly do that.